docs/src/css/codes.css

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: CC-BY-NC-SA-4.0
+ */
+
+.codes .name {
+  cursor: alias;
```

Comment:
I'm not a fan of this cursor. I think it should be replaced in favor of the simple `pointer`. On top of this, clicking isn't obvious as to what it does. We may want a toast triggered on click. This needs to be investigated.

Possibly the addition of `user-select: none` would be nice to also give the impression that it's a copy button (along with a toast).

Reply:
Heh, agreed.  I'm glad someone noticed. I wasn't a fan either but didn't want to waste time on alternatives.  Will look into it!

Reply:
This looks like a possible good candidate for toasts: https://github.com/caroso1222/notyf

Reply:
Gave it a shot with a different library.  What do you think?

Reply:
Looking much better. I'm thinking that `user-select: none` should be added still. Also on Windows the updated cursor just looks really ugly. Not sure why so many Windows cursors have this old jagged style. Other than that, well done!

Reply:
Buy a Mac.